frameworks/elastic/tests/test_tls.py

```diff
@@ -91,3 +117,9 @@ def test_crud_over_tls(elastic_service_tls):
 
     assert document
     assert document['_source']['name'] == 'Loren'
+
+
+@pytest.mark.tls
+@pytest.mark.sanity
+def test_kibana_tls(kibana_application_tls):
+    config.check_kibana_adminrouter_integration("service/{}/login".format(config.KIBANA_PACKAGE_NAME))
```

Comment:
This should be `KIBANA_SERVICE_NAME` right?

Reply:
Yes!